docs/source/getting-started/updating.adoc

```diff
@@ -24,3 +24,5 @@ This command checks whether there is a newer version of Minishift available.
 If so, it prompts user to confirm the update, downloads the new binary and replaces the current version of Minishift.
 
 NOTE: It is recommended to run the `minishift update` command after the xref:../command-ref/minishift_delete.adoc#[`minishift delete`] to avoid backward compatibility issues.
```

Comment:
I think this sentence is confusing and should be changed. I think what we want to say is that one should run _delete_ prior to update. 

"NOTE: We recommend to run xref:../command-ref/minishift_delete.adoc#[`minishift delete`] before running the `update` command to avoid backward compatibility issues.

Reply:
> "NOTE: We recommend to run xref:../command-ref/minishift_delete.adoc#[minishift delete] before running the update command to avoid backward compatibility issues.
+1 this is more clear

Reply:
ok, let;s make the change then.